WasatchNET/Driver.cs

```diff
@@ -376,37 +379,36 @@ void OnUsbError(object sender, UsbError e)
                 if (e.Win32ErrorNumber == 31)
                 {
                     UsbDevice usb = sender as UsbDevice;
-                    if (usb.IsOpen)
+                    if (usb != null && usb.IsOpen)
                     {
                         UsbEndpointBase baseDevice = sender as UsbEndpointBase;
                         // UsbEndpointInfo uei = baseDevice.EndpointInfo;
                         // LibUsbDotNet.Descriptors.UsbEndpointDescriptor ued = uei.Descriptor;
-                        logger.error("{0} [UsbEndPointBase]: usb device still open on endpoint {1}", prefix, baseDevice.EpNum);
-
-                        if (baseDevice.Reset())
-                        {
-                            // docs say to set e.Handled = true here, but UsbError has no such field;
-                            // was commented out here:
-                            // https://github.com/GeorgeHahn/LibUsbDotNet/blob/master/LibWinUsb/UsbDevice.Error.cs#L49
-                            return;
-                        }
+                        logger.error($"{prefix} [UsbEndPointBase]: usb device still open on endpoint 0x{baseDevice.EpNum}, so resetting endpoint");
+                        baseDevice.Reset();

```

Comment:
I'm really hopeful this will fix some of the issues we've been seeing with HOCT. Will have a chance to see tomorrow